test/lib/utils.public.test.js

```diff
@@ -101,6 +101,16 @@ tap.test('prettifyMessage', t => {
     t.is(str, '{30}-30-{30-30}')
   })
 
+  t.test('returns nested value in object', async t => {
+    const str = prettifyMessage({ log: { msg: 'foo', req: { id: '123' } }, messageFormat: '{req.id}' })
+    t.is(str, '123')
+  })
+
+  t.test('returns nested value in object only if not specific key', async t => {
+    const str = prettifyMessage({ log: { msg: 'foo', 'req.id': '567', req: { id: '123' } }, messageFormat: '{req.id}' })
+    t.is(str, '567')
+  })
```

Comment:
can you please add a test where `{object.with.deep.property}` is not present completely?

Reply:
'[object Object]' will get.  Do you have any suggestions for this?

Reply:
I think it should be empty instead

Reply:
return '' instead